sw/device/lib/testing/rstmgr_testutils.c

```diff
@@ -50,13 +50,8 @@ void rstmgr_testutils_compare_cpu_info(
   dif_rstmgr_cpu_info_dump_segment_t
       actual_cpu_dump[DIF_RSTMGR_CPU_INFO_MAX_SIZE];
 
```

Comment:
The checks removed are done in dif_rstmgr_cpu_info_dump_read, and end up with better reporting.